components/ledger/libs/aws/iam/load.go

```diff
@@ -23,6 +24,7 @@ func InitFlags(flags *pflag.FlagSet) {
 	flags.String(AWSSecretAccessKeyFlag, "", "AWS secret access key")
 	flags.String(AWSSessionTokenFlag, "", "AWS session token")
 	flags.String(AWSProfileFlag, "", "AWS profile")
+	flags.String(AWSRoleArnFlag, "", "AWS Role ARN")
```

Comment:
The flag was missing ?

Reply:
Yes this is why I added it

Reply:
Flags are not exhaustive, the are a lot of other potential flags.